Source/Core/DolphinWX/VideoConfigDiag.cpp

```diff
@@ -245,10 +245,13 @@ static wxString dump_textures_desc =
     wxTRANSLATE("Dump decoded game textures to User/Dump/Textures/<game_id>/.\n\nIf unsure, leave "
                 "this unchecked.");
 static wxString load_hires_textures_desc = wxTRANSLATE(
-    "Load custom textures from User/Load/Textures/<game_id>/.\n\nIf unsure, leave this unchecked.");
-static wxString cache_hires_textures_desc =
-    wxTRANSLATE("Cache custom textures to system RAM on startup.\nThis can require exponentially "
-                "more RAM but fixes possible stuttering.\n\nIf unsure, leave this unchecked.");
+    "Load custom textures from User/Load/Textures/<game_id>/.\nDolphin now supports "
+    "DXTn compresssed textures natively, reducing the amount of required RAM by more than 5 times."
+    "\n\nIf unsure, leave this unchecked.");
+static wxString cache_hires_textures_desc = wxTRANSLATE(
+    "Prefetch custom textures to system RAM on startup.\nThis can require exponentially "
+    "more RAM for non-native texture formats such as PNG, but fixes possible stuttering."
```

Comment:
That's what you probably heard from others, they are also wrong if the existing counter is correct.

![gitcontrib_dolphin_customtextures_prefetcheroverhaulcomm1-dxt-dds-memevenlower-nooverhead](https://user-images.githubusercontent.com/20824154/27994275-9ea87b9e-64ba-11e7-8fb4-cfc089e44cb9.png)